watertap/examples/flowsheets/case_studies/wastewater_resource_recovery/amo_1575_hrcs/hrcs.py

```diff
@@ -207,6 +207,7 @@ def add_costing(m):
     )
 
     m.fs.costing = ZeroOrderCosting(case_study_definition=source_file)
+    m.fs.costing.base_currency = pyunits.USD_2018
     m.fs.watertap_costing = WaterTAPCosting()
 
```

Comment:
```suggestion
    m.fs.watertap_costing = WaterTAPCosting()
    m.fs.watertap_costing.base_currency = pyunits.USD_2020

```

Reply:
I know this is the opposite of what we discussed at the time, but setting the base currency on the `watertap_costing` side caused another (non-UI) test to fail, so I just blindly tried to switch the two (15709e45c8097f78d8aabed0893e88c586d2d9d6). If that's not appropriate, I'll try to see if I can reproduce the error to give more accurate information.